src/orga/governance.rst

```diff
@@ -108,68 +109,91 @@ To call for a standard vote, here is a template PR comment:
-         time.  In order for a timeout to occur, the vote must have:
-
-             * been open for at least 2 weeks
-             * been presented and discussed at a Core team meeting
-             * been advertised on at least 3 separate occasions on the gitter core
-               channel (beginning of voting period, middle, and one day prior to
-               proposed timeout)
-             * been sent to core members via email.  Email reminders must have been
-               sent similarly to the gitter core channel: at least 3 times,
-               occurring as beginning of voting period, middle, and one day
-               prior to proposed timeout.
-
-         Extending the above example, if 9 people are required for a quorum, but
-         only 7 have voted, those 7 votes can form the basis of a completed vote
-         after the above conditions are met. 4 votes within those 7 would be
-         needed to pass the vote.
-
-         To post a timeout reminder, here is a template comment:
-
-          .. code-block:: md
-
-              @conda-forge/core
-              This vote falls under the {policy} policy, please vote and/or comment on this PR.
-              This vote needs {policy_percent} of core to vote yea to pass.
-              This vote presently has {current_voters}, and needs {policy_percent * core - current_voters} more for quorum.
-              It is proposed that this vote will time out and be evaluated with the current votes in {days}, on {date}.
-              To vote please leave Approve (yea) or Request Changes (nay) reviews.
-              
-         To declare a standard vote "timed out," the person making such a declaration must post a pull-request adding
-         a vote record to the https://github.com/conda-forge/conda-forge.github.io/tree/master/src/vote-results
-         folder.  The declaration PR should be merged by the first core member who is available to verify that
-         the requirements for the timeout have been met, based on their own personal records.
+
+:Quorum: Quorum for votes can be met in one of three ways depending on the
+     vote: the standard quorum rules, the accelerated quorum rules, and the
+     "time out" quorum rules. The specific quorum rules applicable
+     for each vote are listed below.
+
+     **Standard Quorum Rules**: All percentages below express *both*
+     required participation, as a
+     fraction of the active core team, as well as the fraction of that
+     fraction who vote affirmatively on the issue. For example, in a vote
+     requiring 50%, with 18 active core members, at least 9 must vote;
+     if 9 vote, there must 5 affirmative votes. If 13 members vote, 7
+     must be affirmative.
+
+     **Accelerated Quorum Rules**: For certain votes, we allow a lower quorum level.
+     For these votes,
+     if the vote ends after at least a week and there are no "no" votes, a quorum
```

Comment:
```suggestion
     if the voting period is longer than a week and there are no "no" votes, a quorum
```

Reply:
```suggestion
     if the voting period, which is longer than a week, has concluded and there are no "no" votes, a quorum
```

Reply:
I guess I'm concerned that this clause does 2 things: specifying a minimal length of time needed for the rule to work and what happens after that time. It might be better to separate these clauses into which votes can make use of this rule and when this rule triggers